src/main/kotlin/com/team4099/robot2023/subsystems/falconspin/Motor.kt

```diff
@@ -178,19 +178,19 @@ class Falcon500(
 ) : Motor<Falcon>() {
 
   override val appliedVoltage: ElectricalPotential
-    get() = falcon500.motorOutputVoltage.volts
+    get() = (falcon500.get() * RobotController.getBatteryVoltage()).volts
```

Comment:
I was gonna use supply voltage for this but in phoenix6 code they use battery voltage to scale percent output into volts